### DISCUSSION TITLE
USB no longer recognised under Windows 10 / OpenI6X v1.9.0

### DISCUSSION THREAD

**Discussion by r34dl33** [OpenI6X/opentx#356] [Show and tell]
Hi all, I just finished modding my FS-i6X with OpenI6x OpenTX version 1.9.0. I had to use the ST-Link method, as Windows 10 would not recognised the correct drivers for USB in DFU mode. I also added a USB connector, which I can also power the radio from either my computer or a mobile charging power bank.

Heres the thing, since installing OpenI6X version 1.9.0, the USB connection doesn't find a recognised driver. I get a message in device console saying "device descriptor request failed".

Does anyone else have this issue and / or are there new Windows 10 drivers available, as I'd like to be able to continue using this controller in my Simulators.

![20230516_145423](https://github.com/OpenI6X/opentx/assets/92515879/450f4eca-b95f-4073-aba6-2d4f6f476ef2)
![20230516_145241](https://github.com/OpenI6X/opentx/assets/92515879/b0b4a632-a9d4-4345-a3d4-e6437dbe8fd2)

Note: I connected the USB on the underside of the can and to the original trainer port. This mean I can us the original connectors and when opening the case, I can unplug and remove the back completely. Also, I connected the positive VBUS directly to the battery plus terminal. It works great powering the radio.

**Comment by Davidrenaud314** [comment]
Hi,
I have nearly same issue with 1.9 : Win10 do not detect FS-I6X while STM32 still detect it (in DFU mode). another TX with FW 1.8 was previously recognized, but after plugging the TX 1.9, it is not recognized anymore.
Best regards,
